Sources/SwiftProtobuf/ProtobufBinaryTypes.swift

```diff
@@ -679,6 +750,7 @@ public protocol ProtobufBinaryMessageBase: ProtobufMessageBase {
     // Serialize to protobuf
     func serializeProtobuf() throws -> Data
     func serializeProtobufBytes() throws -> [UInt8]
+    func serializedSize() throws -> Int
```

Comment:
For consistency, this should be `serializedProtobufSize()`  (Yes, I know you think `Protobuf` here is implied.  ;-)